### PR TITLE
FAQ and updates about applications

### PR DESCRIPTION
**Pull request by fayepal**
Transferred the FAQ from Discourse to the website, made some edits and updated with the announcement of the seasonal application

### CODE REVIEW

File: _pages/faq.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+                    </div>
+                </div>
+            </div>
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <h4 class="panel-title">
+                        <a data-toggle="collapse" data-parent="#accordion" href="#collapseTen">What is the relation between the Campus Advisory Committee (CAC) and Regional Coordinators?</a>
+                    </h4>
+                </div>
+                <div id="collapseTen class="panel-collapse collapse">
```
Comment (bacharakis) [inline _pages/faq.md]: Hey @fayetandog ! I think you are missing a " here, and the code breaks.

**Review by bacharakis — COMMENTED** [review]
hey there is a minor bug in line 181.
Let me know as soon as you fix it in order to merge it ;)

**Comment by fayepal** [comment]
Updated! :)

**Comment by bacharakis** [comment]
Awesome! Thank you so much @fayetandog

**Pull request merged**